src/sonic-ctrmgrd/tests/container_test.py

```diff
@@ -0,0 +1,536 @@
+import os
+import pytest
+
+from unittest.mock import MagicMock, patch
+
+from . import common_test
```

Comment:
`pytest` is a third-party lib, `unittest` is part of the standard library. Please re-group imports as follows:

```python3
import os
from unittest.mock import MagicMock, patch

import pytest

from . import common_test
```